<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Perspective</class>
 <widget class="QDialog" name="Perspective">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置透视</string>
  </property>
  <widget class="QLabel" name="lblPSPBGN">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblPicture">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>175</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblPosLU">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>175</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>左上坐标：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePosLU">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>225</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblPosRU">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>275</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>右上坐标：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePosRU">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>325</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblPosLD">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>375</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>左下坐标：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePosLD">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>425</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblPosRD">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>475</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>右下坐标：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePosRD">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>525</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>675</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>确认坐标</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRevoke">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>600</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>清除坐标</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnRevoke</sender>
   <signal>clicked()</signal>
   <receiver>Perspective</receiver>
   <slot>OnRevoke()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>900</x>
     <y>608</y>
    </hint>
    <hint type="destinationlabel">
     <x>880</x>
     <y>-19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnConfirm</sender>
   <signal>clicked()</signal>
   <receiver>Perspective</receiver>
   <slot>OnConfirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>906</x>
     <y>699</y>
    </hint>
    <hint type="destinationlabel">
     <x>776</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnRevoke()</slot>
  <slot>OnConfirm()</slot>
 </slots>
</ui>
